pbsmetrics/prometheus/prometheus_test.go

```diff
@@ -0,0 +1,423 @@
+package prometheusmetrics
+
+import (
+	"regexp"
+	"strconv"
+	"testing"
+	"time"
+
+	"github.com/prometheus/client_golang/prometheus"
+	dto "github.com/prometheus/client_model/go"
+
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/openrtb_ext"
+	"github.com/prebid/prebid-server/pbsmetrics"
+)
+
+var gaugeValueRegexp = regexp.MustCompile("gauge:<value:([0-9]+) >")
+var counterValueRegexp = regexp.MustCompile("counter:<value:([0-9]+) >")
+var histogramValueRegexp = regexp.MustCompile("histogram:<sample_count:([0-9]+)")
+
+func TestConnectionMetrics(t *testing.T) {
+	proMetrics := newTestMetricsEngine().(*Metrics)
```

Comment:
Can you get rid of all these casts if you just change the return type on `newTestMetricsEngine` to `*Metrics`?